common/changes/@microsoft/api-extractor/trim-export-default-from-reports_2025-04-29-00-30.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Correctly omit declarations associated with an entity whose release tag does not meet the configured threshold in API reports",
```

```suggestion
      "comment": "Fix an issue where default exports were sometimes trimmed incorrectly in .api.md files when using `reportVariants` (GitHub #4775)",
```

The original phrasing maybe "buried the lead" a bit with _"in API reports"_. I had to reread it a couple times to understand that this wasn't about .d.ts rollups. 😆